bootstrap/startup.json.default

```diff
@@ -18,6 +18,10 @@
                 "bind": "/tmp/wpa_playground",
                 "mode": "rw"
             }
+            "/root/.config/companion-core": {
+                "bind": "/root/.config",
+                "mode": "rw"
+            }
```

Comment:
Just to be sure, this would be for companion-core general settings, right?
Because the settings for the different services are currently under `/roor/config/{service.name}`

Reply:
yep, but this was the wrong approach as willian comment points:
https://github.com/bluerobotics/companion-docker/pull/120#issuecomment-804425202